packages/graphql-connection-transformer/src/__tests__/ModelConnectionTransformer.test.ts

```diff
@@ -93,6 +93,31 @@ test('Test ModelConnectionTransformer simple one to many happy case with custom
   expect(connectionUpdateId).toBeTruthy();
 });
 
+test('Test that ModelConnection Transformer throws error when the field in connection is not found in the related Type', () => {
+  const invalidSchema = `
+  type Post @model {
+    name: String!
+    teamID: ID!
+    team: Team @connection(fields: ["teamID"])
+  }
+
+  type Team @model {
+    name: [String!]!
+  }
+  `;
+  const transformer = new GraphQLTransform({
+    transformers: [new DynamoDBModelTransformer(), new ModelConnectionTransformer()],
+  });
+  try {
+    transformer.transform(invalidSchema);
+    expect(true).toEqual(false);
```

Comment:
nit could use `expect(() => transformer.transform(invalidSchema)).toThrow(...)`

Reply:
toThrow expects an error to be thrown. I tried using `toThrowException` to expect a specific error though it doesn't seem like jest is picking it up.